.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I added this because by default dart formats to 80 characters which caused a lot of changes. We should either .gitignore this file or keep it so that contributers don't have to swap back and forth their line length when switching projects.

Reply:
Then gitignore, I just want to avoid ide specific files in the project